en/log/flight_log_analysis.md

```diff
@@ -1,10 +1,6 @@
 # Flight Log Analysis
 
-This topic outlines approaches and software packages that can be used to analyze PX4 flight logs.
-
-## Reporting Flights
-
-[Flight Reporting](../getting_started/flight_reporting.md) explains how to download a log and report/discuss issues about a flight with the development team.
```

Comment:
While nothing to do with log analysis, this is the most important thing in the page :-). We need it back in some form or another.